src/components/views/verification/TermsAndConditions.tsx

```diff
@@ -33,14 +33,30 @@ export default function TermsAndConditions() {
 					},
 				},
 			});
-			setVerificationData(data.updateProjectVerificationForm);
+			setVerificationData(prevState =>
+				prevState
+					? {
+							...prevState,
+							status: EVerificationStatus.DRAFT,
+					  }
+					: undefined,
+			);
+
 			setloading(false);
 			setStep(8);
 		}
 
 		if (isChanged && isDraft) {
 			sendReq();
 		} else {
+			setVerificationData(prevState =>
+				prevState
+					? {
+							...prevState,
+							status: EVerificationStatus.DRAFT,
+					  }
+					: undefined,
+			);
```

Comment:
I suggest you have a function and call it to avoid repeating the code. (I know it's not a lot of repetition, but we can reduce the number of lines and this is a suggestion)
Thanks